core/server/master/src/main/java/alluxio/master/AlluxioMasterProcess.java

```diff
@@ -261,6 +260,7 @@ private void startServing() {
    */
   protected void startServing(String startMessage, String stopMessage) {
     MetricsSystem.startSinks();
+    mSafeModeManager.notifyRpcServerStarted();
```

Comment:
we should do this immediately before calling `mThriftServer.serve()` in `startServingRPCServer`

Reply:
Done.